sdk-setup/src/sdk-manage

```diff
@@ -611,6 +611,20 @@ object_upgradable() {
     object_zypper "$type:$object" --quiet --no-refresh list-updates
 }
 
+rebuild_rpmdb() {
+    local object=$1
+    # rpm can't overwrite directory on a different overlay layer
+    if [[ $(findmnt --noheadings --output FSTYPE / ) == overlay ]]; then
+        enter "$object" rpm --rebuilddb &>/dev/null
+        local new_db=$(enter "$object" find /var/lib -maxdepth 1 -name 'rpmrebuilddb.*' |head -n1)
+        if [[ $new_db ]]; then
+            enter "$object" sh -c "rm -rf /var/lib/rpm && mv $new_db /var/lib/rpm"
```

Comment:
Worth a comment that in case of targets we need to do those under single sb2 session and why.